src/components/networked.js

```diff
@@ -93,45 +108,19 @@ AFRAME.registerComponent('networked', {
     NAF.entities.registerEntity(networkId, this.el);
   },
 
-  initTemplate: function(template, show) {
-    if (this.templateEl) {
-      this.el.removeChild(this.templateEl);
-    }
-
-    var templateChild = document.createElement('a-entity');
-    templateChild.setAttribute('template', 'src:' + template);
-    templateChild.setAttribute('visible', show);
-
-    this.el.appendChild(templateChild);
-    this.templateEl = templateChild;
-  },
-
   firstUpdate: function() {
     var entityData = this.el.firstUpdateData;
-    this.networkUpdate(entityData); // updates root element only
-
-    this.waitForTemplate(() => {
-      this.networkUpdate(entityData);
-    });
-  },
-
-  waitForTemplate: function(callback) {
-    // wait for template to render then callback
-    if (this.templateEl) {
-      this.templateEl.addEventListener('templaterendered', function() { setTimeout(callback); });
-    } else {
-      setTimeout(callback);
-    }
+    this.networkUpdate(entityData);
   },
 
   onConnected: function() {
     if (this.data.owner === '') {
       this.lastOwnerTime = NAF.connection.getServerTime();
       this.el.setAttribute(this.name, {owner: NAF.clientId});
-
-      this.waitForTemplate(() => {
+      setTimeout(() => {
+        //a-primitives attach their components on the next frame; wait for components to be attached before calling syncAll
```

Comment:
I've added an issue for this. We shouldn't need to do this.
https://github.com/aframevr/aframe/issues/3431